exchanges/huobi/huobi_test.go

```diff
@@ -1981,489 +1223,305 @@ func TestCancelAllExchangeOrders(t *testing.T) {
 }
 
-func TestWsUnsubscribe(t *testing.T) {
-	pressXToJSON := []byte(`{
-  "id": "id4",
-  "status": "ok",
-  "unsubbed": "market.btcusdt.trade.detail",
-  "ts": 1494326028889
-}`)
-	err := h.wsHandleData(pressXToJSON)
-	if err != nil {
-		t.Error(err)
-	}
+func TestWithdrawInternationalBank(t *testing.T) {
+	t.Parallel()
+	_, err := h.WithdrawFiatFundsToInternationalBank(context.Background(), &withdraw.Request{})
+	assert.ErrorIs(t, err, common.ErrFunctionNotSupported)
 }
 
-func TestWsKlineArray(t *testing.T) {
-	pressXToJSON := []byte(`{
-  "status": "ok",
-  "rep": "market.btcusdt.kline.1min",
-  "data": [
-    {
-      "amount": 1.6206,
-      "count":  3,
-      "id":     1494465840,
-      "open":   9887.00,
-      "close":  9885.00,
-      "low":    9885.00,
-      "high":   9887.00,
-      "vol":    16021.632026
-    },
-    {
-      "amount": 2.2124,
-      "count":  6,
-      "id":     1494465900,
-      "open":   9885.00,
-      "close":  9880.00,
-      "low":    9880.00,
-      "high":   9885.00,
-      "vol":    21859.023500
-    }
-  ]
-}`)
-	err := h.wsHandleData(pressXToJSON)
-	if err != nil {
-		t.Error(err)
+func TestQueryDepositAddress(t *testing.T) {
+	t.Parallel()
+	_, err := h.QueryDepositAddress(context.Background(), currency.USDT)
+	if sharedtestvalues.AreAPICredentialsSet(h) {
+		require.NoError(t, err)
+	} else {
+		require.ErrorIs(t, err, exchange.ErrCredentialsAreEmpty)
 	}
 }
 
-func TestWsMarketDepth(t *testing.T) {
-	pressXToJSON := []byte(`{
-  "ch": "market.htusdt.depth.step0",
-  "ts": 1572362902027,
-  "tick": {
-    "bids": [
-      [3.7721, 344.86],
-      [3.7709, 46.66]
-    ],
-    "asks": [
-      [3.7745, 15.44],
-      [3.7746, 70.52]
-    ],
-    "version": 100434317651,
-    "ts": 1572362902012
-  }
-}`)
-	err := h.wsHandleData(pressXToJSON)
-	if err != nil {
-		t.Error(err)
+func TestGetDepositAddress(t *testing.T) {
+	t.Parallel()
+	_, err := h.GetDepositAddress(context.Background(), currency.USDT, "", "uSdTeRc20")
+	if sharedtestvalues.AreAPICredentialsSet(h) {
+		require.NoError(t, err)
+	} else {
+		require.ErrorIs(t, err, exchange.ErrCredentialsAreEmpty)
 	}
 }
 
-func TestWsBestBidOffer(t *testing.T) {
-	pressXToJSON := []byte(`{
-	  "ch": "market.btcusdt.bbo",
-	  "ts": 1489474082831,
-	  "tick": {
-		"symbol": "btcusdt",
-		"quoteTime": "1489474082811",
-		"bid": "10008.31",
-		"bidSize": "0.01",
-		"ask": "10009.54",
-		"askSize": "0.3"
-	  }
-	}`)
-	err := h.wsHandleData(pressXToJSON)
-	if err != nil {
-		t.Error(err)
+func TestQueryWithdrawQuota(t *testing.T) {
+	t.Parallel()
+	_, err := h.QueryWithdrawQuotas(context.Background(), currency.BTC.Lower().String())
+	if sharedtestvalues.AreAPICredentialsSet(h) {
+		require.NoError(t, err)
+	} else {
+		require.ErrorIs(t, err, exchange.ErrCredentialsAreEmpty)
 	}
 }
 
+func TestWsCandles(t *testing.T) {
+	h := new(HUOBI) //nolint:govet // Intentional shadow to avoid future copy/paste mistakes
```

Comment:
Any reason these can't be parallel given that they are shadowed and use their own Json files?

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/58ecd3c1b